devdoc/rc_string_requirements.md

```diff
@@ -72,6 +72,33 @@ MOCKABLE_FUNCTION(, THANDLE(RC_STRING), rc_string_create, const char*, string);
 
 **SRS_RC_STRING_01_006: [** If any error occurs, `rc_string_create` shall fail and return `NULL`. **]**
 
+## rc_string_create_with_format
+
+```c
+THANDLE(RC_STRING) rc_string_create_with_format(const char* format, ...);
+```
+
+`rc_string_create_with_format` creates a new ref counted string by using the format convention as in sprintf.
+
+If `format` is `NULL`, `rc_string_create_with_format` shall fail and return `NULL`. 
+
+Otherwise, `rc_string_create_with_format` shall determine the total number of characters written using the variadic arguments. 
```

Comment:
>variadic [](http://example.com/codeflow?start=107&length=8)

this word actually does not appear in the C99 or C11 standard.
The standard way of talking about these is variable number of arguments